### DISCUSSION TITLE
Are v-row and v-col mandatory inside v-container?

### DISCUSSION THREAD

**Discussion by HendrikJan** [vuetifyjs/vuetify#14410] [Q&A] [answered]
I want to place all my elements on a page below each other, with a bit of spacing between.
Elements like `v-simple-table`, `v-btn`, `<h2>`.

I can do that inside a v-container like this:
```html
<v-container>
  <h2>This is my webpage</h2>
  <v-simple-table>...</v-simple-table>
  <v-btn>Save</v-btn>
</v-container>
```
Which looks clean to me.

Or I can do it like this:
```html
<v-container>
  <v-row>
    <v-col>
      <h2>This is my webpage</h2>
    </v-col>
  </v-row>
  <v-row>
    <v-col>
      <v-simple-table>...</v-simple-table>
    </v-col>
  </v-row>
  <v-row>
    <v-col>
      <v-btn>Save</v-btn>
    </v-col>
  </v-row>
</v-container>
```
Which is much more cumbersome and doesn't look as nice.

So I am wondering, what do I gain by using `v-row` and `v-col` everywhere?

**Comment by KaelWD** [comment, marked as answer]
Nope. row/col without any sizing props mostly just add gutters, if you don't want gutters you don't need them.

**Reply by HendrikJan** [reply]
Thanks.
I'll go without rows and cols than.
Using `class="mx-2"` instead to create some spacing between the elements.
